### PR TITLE
Feature/calendar api

### PR DESCRIPTION
**Pull request by santee**
@serjKim just a first idea for days off. I've added counters directly to employee model, couldn't come up with a better idea, so you need to request it every time after you submit a dayoff and recieve a successful response

### CODE REVIEW

File: server/Arcadia.Assistant.Web/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -75,6 +75,8 @@ public EmployeesController(IActorRefFactory actorSystem, ActorPathsBuilder paths
                         var employee = EmployeeModel.FromMetadata(x.Metadata);
                         var photo = await x.Actor.Ask<GetPhoto.Response>(GetPhoto.Instance, timeout, token);
                         employee.Photo = photo.Photo;
+                        employee.HoursCredit = 12;
```
Comment (serjKim) [inline server/Arcadia.Assistant.Web/Controllers/EmployeesController.cs]: Did you forget to commit this property?

Comment (santee) [reply server/Arcadia.Assistant.Web/Controllers/EmployeesController.cs]: no, the property is committed, name change didn't pass through though...

Comment (santee) [reply server/Arcadia.Assistant.Web/Controllers/EmployeesController.cs]: fixed

File: server/Arcadia.Assistant.Web/Models/EmployeeModel.cs [hunk 2]
```diff
@@ -26,6 +26,14 @@ public class EmployeeModel
 
         public DateTime? HireDate { get; set; }
 
+        public int VacationDaysLeft { get; set; }
+
+        /// <summary>
+        /// Positive values means that these days must be worked out.
+        /// Negative means that these can be taked as days off
+        /// </summary>
+        public int HoursCredit { get; set; }
```
Comment (serjKim) [inline server/Arcadia.Assistant.Web/Models/EmployeeModel.cs]: Maybe versa vice? positive means days off, negative - work out?

**Pull request merged**